packages/boilerplate-generator-tests/web.browser-tests.js

```diff
@@ -39,3 +39,24 @@ Tinytest.add("boilerplate-generator-tests - web.browser - call rewriteHook", fun
 Tinytest.add("boilerplate-generator-tests - web.browser - include runtime config", function (test) {
   test.matches(html, /<script[^<>]*>[^<>]*__meteor_runtime_config__ =.*decodeURIComponent\(config123\)/);
 });
+
+// https://github.com/meteor/meteor/issues/9149
+Tinytest.add(
+  "boilerplate-generator-tests - web.browser - properly render boilerplate " +
+  "elements when _.template settings are overridden",
+  function (test) {
+    import { _ } from 'meteor/underscore';
```

Comment:
Should `boilerplate-generator-tests` be depending on `underscore` in its `package.js` via `api.use`?

Reply:
Yes - I noticed that as well and have added it into my forthcoming changes. Thanks!

Reply:
Change pushed - I've also removed `_` wherever possible.